packages/docs/src/components/header/header.tsx

```diff
@@ -68,6 +68,22 @@ export const Header = component$(() => {
               <span>Qwik City</span>
             </a>
           </li>
+          <li>
+            <a
+              href="/deployments/overview/"
```

Comment:
I am concerned that the top level nav-bar will become too busy. What do you think about merging the `deployments` and `Integrations` into a single section. 

So top level would be `Integrations`

and than on the sidebar it would be Deployments/Integration/3rd-party/etc...

Reply:
Yes, it's overcrowded now. will merge this

Reply:
@mhevery merged into integrations. Please see the screenshot above.